src/mscorlib/src/System/Diagnostics/Eventing/TraceLogging/FieldMetadata.cs

```diff
@@ -59,44 +59,6 @@ internal class FieldMetadata
             return;
         }
 
-        /// <summary>
-        /// Fixed-length array.
-        /// </summary>
-        public FieldMetadata(
-            string name,
-            TraceLoggingDataType type,
-            EventFieldTags tags,
-            ushort fixedCount)
-            : this(
-                name,
```

Comment:
removed method. I'm relying on your implying that desktop doesn't need it either.